modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -30,19 +31,16 @@ An API fragment is a RAML document that has a version and an identifier, but is
 == Procedure
 
 . On the *Projects* page in Design Center, click *Create* and select *Create Fragment*.
-. In the *New API Fragment* dialog, type a name for the project and select the type of API fragment that you want to create. Then, click *Create*.
+. In the *New Fragment* dialog, type a name for the project. You can change the name (but not the extension) later, if you want to.
+. Select the type of API fragment that you want to create. Then, click *Create Fragment*.
```

Comment:
Limit steps to one sentence (and one action, if complicated), or create separate steps:
```suggestion
. Select the type of API fragment that you want to create and click *Create Fragment*.
```